site/documentation/how-do-i-write-policies/index.md

```diff
@@ -786,6 +786,34 @@ undefined
 
 In some cases, having an undefined result for a document is not desirable. In those cases, policies can use the [Default Keyword](#default) to provide a fallback value.
 
+## <a name="functions"></a> Functions
+
+Rego supports user defined functions. Functions are only accessible within the modules they are defined in. Once defined, a user function behaves like a builtin.
```

Comment:
Agreed.  Function builtins are a special case of virtual docs in that we deem some arguments 'inputs' and all inputs to a function/builtin must be provided before evaluating.  We use a different notation for invocation and definition to signify that difference.  But otherwise, they should be the same.